src/engine/story-utils.ts

```diff
@@ -0,0 +1,16 @@
+import { Engine, DisplayMode } from '.';
+
+/**
+ * Helper to generate Storybook game engine instance
+ * @param storyFn The storybook fn to pass the engine to
+ */
+export const withEngine = (storyFn: (game: Engine) => void) => {
+  return () => {
```

Comment:
This will get invoked every time story re-renders or HMR reloads modules. It did expose some issues with Ex where we e.g. don't clean up the Play button in the DOM.

Reply:
Uff, yah. We probably should open an issue to clean up the Play button after the game has started.